lib/generate.js

```diff
@@ -68,6 +68,10 @@ function transformValue(value, field, spec) {
 				value = value.replace(nonAsciiRegex, '*');
 			if (!spec.supportsNewLineChars)
 				value = value.replace(newLineRegex, ' ');
+			if (!spec.supportsExtraRecordEndingChars) {
+				var recordEndingRegex = new RegExp(`[${spec.recordEnding}]`, 'g');
```

Comment:
`recordEnding` can be more than one character, so wrapping in square brackets would not exhibit the expected behaviour